api/staged_products_property_collection_guid_assigner_test.go.go

```diff
@@ -9,17 +10,25 @@ import (
 )
 
 var _ = Describe("ResponsePropertyCollection", func() {
-	unmarshalJSON := func(json string) interface{} {
+	unmarshalJSONLikeApiGetStagedProductProperties := func(json string) interface{} {
```

Comment:
Out of interest; why does [`Api.GetStagedProductProperties`](https://github.com/pivotal-cf/om/blob/d17b3b7de0e3c435ea14951bb05dbc0616b95a93/api/staged_products_service.go#L323) use `yaml.Unmarshal()` rather than `json.Unmarshal()`?

Reply:
No particular reason. We inherited this codebase from the releng team, and haven't made everything consistent yet

Reply:
Good to know.  

My 2c - we should switch to `json.Unmarshal`; since it produces more usable `map[string]interface{}` interfaces.